engineapi/alembic/versions/4f05d212ea49_live_at_for_metatx.py

```diff
@@ -0,0 +1,28 @@
+"""Live at for metatx
+
+Revision ID: 4f05d212ea49
+Revises: 040f2dfde5a5
+Create Date: 2023-10-03 10:00:09.730620
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '4f05d212ea49'
+down_revision = '040f2dfde5a5'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('call_requests', sa.Column('live_at', sa.DateTime(timezone=True), server_default=sa.text("TIMEZONE('utc', statement_timestamp())"), nullable=False))
```

Comment:
`live_at` should be nullable. Default can be `NULL`.

Reply:
It will require to add additional logic which I would prefer to not do if the result will be same - just return list of call_requests